code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -287,7 +287,7 @@
 				var/datum/mind/targeted =  A.find_target()//easy way, i dont feel like copy pasting that entire block of code
 				if(!targeted)
 					break
-				targets[targeted.current.real_name] = targeted.current
+				targets["[targeted.current.real_name] the [targeted.assigned_role]"] = targeted.current
```

Comment:
is it possible for them to not have a role?

Reply:
i dont think they can be targeted if they dont have a role in the first place

Reply:
I'm pretty sure it will show up as X the unassigned or something